src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -1,42 +1,51 @@
-import React from 'react';
+import React, { useRef } from 'react';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { useHistory } from 'react-router-dom';
-
+import ReCAPTCHA from 'react-google-recaptcha';
 import './RegisterForm.css';
 import RegisterFormData from '../../interfaces/RegisterFormData';
 import { registerUser, logIn } from '../../lib/auth';
 import { useUserContext } from '../../contexts/UserContext';
+import Recaptcha from '../Recaptcha/Recaptcha';
 
 export default function RegisterForm(): JSX.Element {
     const history = useHistory();
     const user = useUserContext();
+    const recaptchaRef = useRef<ReCAPTCHA>(null);
+
     React.useEffect(() => {
         if (user) {
             // eslint-disable-next-line
             console.log('User already logged in, redirecting to home page');
             history.push('/');
         }
     });
-    const registerHandler = (e: React.FormEvent<HTMLFormElement>) => {
+
+    const registerHandler = async (e: React.FormEvent<HTMLFormElement>) => {
         e.preventDefault();
         const formData = new FormData(e.target as HTMLFormElement);
         const formValue: RegisterFormData = Object.fromEntries(
             formData.entries()
             // eslint-disable-next-line
         ) as any;
         const { email, password } = formValue;
-        registerUser(formValue)
-            .then(async ({ data }) => {
-                // eslint-disable-next-line
-                console.log(`User successfully created!`, data);
-                await logIn(email, password);
-                history.push('/');
-            })
+        // eslint-disable-next-line
+        console.log(formValue);
```

Comment:
Was this just for testing? Can we remove it?

Reply:
I can't remember the exact reason why it's there, but @navarrojandg  was testing it within the browser's console and it was receiving an error there, but technically the code compiles and runs fine with or without it. Jan what do you think?

Reply:
Yeah this is an unnecessary comment, will remove. Was just being used to verify that the recaptcha was being executed and appended to the form.